src/platform/forms-system/src/js/fields/TitleField.jsx

```diff
@@ -1,18 +1,49 @@
 import React from 'react';
+
+import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-export default function TitleField({ id, title }) {
+export default function TitleField({ id, title, formContext }) {
```

Comment:
That's cool that we have control to update this part, I didn't realize that. the solution looks smart, but I'm thinking what if the team actually wants to show an h3 as an h3 on the review page? I'm sure there are probably use cases like that.
I'm hesitant of the idea of making this default behavior for every form because of retroactively affecting the existing forms, and also the idea that some teams may actually want the styling to be the same.
Also TitleField would probably affect every 'ui:title' situation, including arrays, or maybe some individual components/widgets using 'ui:title'

What if instead we allowed to modify on a case-by-case basis? but I thought we could already achieve this with ui:reviewField ?  but that doesn't work in this situation?

Reply:
I can't think of why any consumer would intentionally re-introduce the a11y-bug by duplicating the Review-page accordion H3.  The Review-page accordions are hard-coded H3s, so we're always gonna need the page's TitleField to be H4s on the Review page.

Reply:
That's a great point. I didn't fully understand but do now. thanks.  Hmm but I think maybe not all teams use that specific array collapsible for their ending page.